docs/sphinx/dev_guide/build_configurations.rst

```diff
@@ -0,0 +1,151 @@
+RAJA Build Scripts
+===================
+
+The build scripts in the RAJA ``scripts`` directory are used mostly by RAJA 
+developers to quickly create a build environment to compile and run tests
+during code development. 
+
+Each script can be executed from the top-level RAJA directory. When a script
+is run, it creates a uniquely-named build directory in RAJA and runs CMake 
+with arguments contained in the script to create a build environment in the
+new directory. One then goes into the directory and runs make to build RAJA, 
+its tests, example codes, etc.  For example,
+
+.. code-block:: bash
+
+  $ ./scripts/lc-builds/toss3_clang10.0.1.sh
+  $ cd build_lc_toss3-clang-10.0.1
+  $ make -j
+  $ make test
+
+Eventually, these scripts may go away and be superceded by the Spack-based
+host-config file generation process when that achieves the level of
+compiler coverage that the scripts have.
+
+
+============================
+Generated Host-Config Files
+============================
+
+The RAJA repository contains two submodules 
+`uberenv <https://github.com/LLNL/uberenv>`_ and
+`radiuss-spack-configs <https://github.com/LLNL/radiuss-spack-configs>`_ that 
+work together to generate host-config files. These are projects in the 
+GitHub LLNL organization and contain utilities shared by various projects. 
+The main uberenv script can be used to drive Spack to generate a *host-config* 
+file that contains all the information required to define a RAJA build 
+environment. The host-config file can then be passed to CMake using the '-C' 
+option to create a build configuration. *Spack specs* defining compiler 
+configurations are maintained in files in the radiuss-spack-configs 
+repository.
+
+RAJA shares its uberenv workflow with other projects. The documentation 
+for this is available in `RADIUSS Uberenv Guide <https://radiuss-ci.readthedocs.io/en/latest/uberenv.html#uberenv-guide>`_.
+
+
+Generating a RAJA host-config file
+------------------------------------
+
+This section describes the host-config file generation process for RAJA.
+
+Machine specific configurations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Compiler configurations for Livermore computer platforms are contained in 
+in sub-directories in the RAJA ``scripts/uberenv/spack_configs`` directory:
+
+.. code-block:: bash
+
+  $ ls -c1 ./scripts/uberenv/spack_configs
+  blueos_3_ppc64le_ib
+  darwin
+  toss_3_x86_64_ib
+  blueos_3_ppc64le_ib_p9
+  config.yaml
+
+To see currently supported configurations, please see the contents of the 
+``compilers.yaml`` file in each of these sub-directories.
+
+Generating a host-config file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The main uberenv python script can be invoked from the top-level RAJA directory
+to generate a host-config file for a desired configuration. For example,
+
+.. code-block:: bash
+
+  $ python ./scripts/uberenv/uberenv.py --spec="%gcc@8.1.0"
+  $ python ./scripts/uberenv/uberenv.py --spec="%gcc@8.1.0~shared+openmp tests=benchmarks"
```

Comment:
Add a note that other supported spec's can be found in .gitlab/{LC_machine_name}-jobs.yml?

Reply:
Thanks. Added a reference to the vetted specs section in the CI part.